src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './UserInfo.scss';
+
+type UserProps = {
+  user: User | undefined,
+};
+
+export const UserInfo: React.FC<UserProps> = ({ user }) => (
+  <div className="user__info">
+    <p className="user__name">{user?.username}</p>
+    <p>{user?.email}</p>
```

Comment:
check in the parent component  if user is not null, no need to render empty HTML elements